exporter/googlecloudpubsubexporter/config_test.go

```diff
@@ -29,16 +28,15 @@ import (
 
 func TestLoadConfig(t *testing.T) {
 	factories, err := componenttest.NopFactories()
-	assert.Nil(t, err)
+	assert.NoError(t, err)
```

Comment:
This should be `require` since it shouldn't allow the test to progress if this fails.